src/event-handler.ts

```diff
@@ -1,19 +1,64 @@
 import { Webhooks } from "@octokit/webhooks";
 import { Context, SimplifiedContext } from "./context";
+import { customOctokit } from "./octokit";
+
+export type Options = {
+  webhookSecret: string;
+  appId: string | number;
+  privateKey: string;
+};
 
 export class EventHandler {
   public webhooks: Webhooks<SimplifiedContext>;
   public on: Webhooks<SimplifiedContext>["on"];
   public onAny: Webhooks<SimplifiedContext>["onAny"];
   public onError: Webhooks<SimplifiedContext>["onError"];
 
-  constructor(secret: string) {
+  private _webhookSecret: string;
+  private _privateKey: string;
+  private _appId: number;
+
+  constructor(options: Options) {
+    this._privateKey = options.privateKey;
+    this._appId = Number(options.appId);
+    /*const key = crypto.subtle
+      .importKey(
+        "pkcs8",
+        new TextEncoder().encode(this.privateKey),
+        {
+          name: "RSASSA-PKCS1-v1_5",
+          hash: "SHA-256",
+        },
+        true,
+        []
+      )
+      .then((key) => {
+        crypto.subtle.exportKey("s", key).then((keydata) => {
+          console.log(keydata);
+        });
+      });*/
```

Comment:
What is this comment?

Reply:
It's from when I was trying to convert PKCS#1 to PKCS#8 using crypto module

Reply:
Should this be removed and instead we handle it from the shell?